mapie/tests/test_conformity_scores.py

```diff
@@ -1,35 +1,55 @@
 import numpy as np
 import pytest
 
-from mapie._typing import ArrayLike, NDArray
+from mapie._typing import NDArray, ArrayLike
 from mapie.conformity_scores import (AbsoluteConformityScore, ConformityScore,
                                      GammaConformityScore)
+from mapie.regression.estimator import EnsembleRegressor
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import KFold
 
-X_toy = np.array([0, 1, 2, 3, 4, 5]).reshape(-1, 1)
-y_toy = np.array([5, 7, 9, 11, 13, 15])
-y_pred_list = [4, 7, 10, 12, 13, 12]
-conf_scores_list = [1, 0, -1, -1, 0, 3]
-conf_scores_gamma_list = [1 / 4, 0, -1 / 10, -1 / 12, 0, 3 / 12]
+X_toy_train = np.array([0, 1, 2, 3, 4, 5]).reshape(-1, 1)
+y_toy_train = np.array([5, 7, 9, 11, 13, 15])
+X_toy_test = np.array([6, 9, 10, 2, 4, 5]).reshape(-1, 1)
+y_toy_test = np.array([15, 4, 90, 1, 15, 1])
+y_pred_list = [17., 23., 25.,  9., 13., 15.]
+conf_scores_list = [-2., -19., 65., -8., 2., -14.]
+conf_scores_gamma_list = [-0.11764706, -0.82608696,  2.6,
+                          -0.88888889,  0.15384615, -0.93333333]
+random_state = 42
 
 
 class DummyConformityScore(ConformityScore):
     def __init__(self) -> None:
         super().__init__(sym=True, consistency_check=True)
 
     def get_signed_conformity_scores(
-        self, y: ArrayLike, y_pred: ArrayLike,
+        self, X: ArrayLike, y: ArrayLike, y_pred: ArrayLike,
     ) -> NDArray:
         return np.subtract(y, y_pred)
 
     def get_estimation_distribution(
-        self, y_pred: ArrayLike, conformity_scores: ArrayLike
+        self, X: ArrayLike, y_pred: ArrayLike, values: ArrayLike
```

Comment:
UPD: change `values` to `conformity_scores`

Reply:
Done